speedometer-reusable/js/speedometer.js

```diff
@@ -158,7 +160,9 @@ $.fn.myfunc = function (userPref) {
       "-moz-transform"    :'rotate('+speedInDeg+'deg)',
       "-o-transform"      :'rotate('+speedInDeg+'deg)'
     });
-    self.parentElem.find(".speedPosition").html(speed);
+    
+    centerval = speed *  self.defaultProperty.multiplier;
```

Comment:
Please define scope for centerval (add "var"), also convert centerval => center**V**al